src/ComputeSharp.Graphics/Buffers/ConstantBuffer{T}.cs

```diff
@@ -44,31 +44,20 @@ private static int GetPaddedSize()
         public T this[int i] => throw new InvalidExecutionContextException($"{nameof(ConstantBuffer<T>)}<T>[int]");
 
         /// <inheritdoc/>
-        public override void GetData(Span<T> span, int offset, int count)
+        public override unsafe void GetData(Span<T> span, int offset, int count)
         {
             if (IsPaddingPresent)
```

Comment:
You can move the:

```csharp
using MappedResource resource = MapResource();
```

At the start of the method now, as it's used in both branches for the entire duration of their scope.
You can also make the `else` block inline at that point, as it'll only have a single statement.

Reply:
Done!